### ISSUE TITLE
Unable synthesize/clone in own voice

### ISSUE THREAD

**Issue by chor-developer** [CorentinJ/Real-Time-Voice-Cloning#19]
Hi,

I am new to pytorch and audio related manipulations using python. I have installed and configured along with sample train-clean-100 dataset. Now I am able to record 5-7 samples of my own voice but the for some reason they synthesize is not happening in my own voice. it seems to be doing in a default male voice. Can you please guide me in right direction? 

Thanks for the help.

**Comment by CorentinJ** [comment]
Do you hear your own voice after recording it? Does the recording sound good?

**Comment by chor-developer** [comment]
Yes, i can hear my own voice nd other histogram data of the samples are also getting created.

**Comment by CorentinJ** [comment]
If you're doing everything right, then it's just the ability of the model to clone a voice that is failing you. If you look at page 8 in the middle of [the SV2TTS paper](https://arxiv.org/pdf/1806.04558.pdf), you will see that the mean opinion score for voice resemblance is of 3 (where the min score is 1 and the max is 5). The framework has its limitations. For voices out of the distribution, it's not surprising to see poor performance. I noticed that for my voice, for example, I had to really make an effort to have a proper English accent (my native language is French) to get a voice somewhat close to mine. I also have all the datasets downloaded (VoxCeleb & LibriSpeech), and while most of the time the voice cloning is good (even on the test set of voxceleb, which has crappy quality), there are still a few uncommon voices for which the framework doesn't perform so well, and defaults to an "average voice" instead.

There isn't much I can do about that, these are the limitations of the SV2TTS framework and only new findings will allow for improvements in the domain of voice cloning.
